.github/workflows/verifybuild.yml

```diff
@@ -0,0 +1,21 @@
+name: Java CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
Hi @dvlato. When searching the documentation, it is always checkout@v2 (https://github.com/actions/checkout) that comes out at the top. 

Therefore asking, how did you come about using checkout@v1? Do you know if the v1 and v2 are related in any way? Should be using the v2 since it is newer? Etc.

Reply:
Okay I see, the examples for Setup-java are using checkout@v1. (https://github.com/actions/setup-java).

Reply:
And here: https://github.com/actions/cache/blob/master/examples.md#java---maven

Reply:
see the 'maven guide' was updated from checkout@v1 to @v2 around Christmas time:   https://github.com/actions/starter-workflows/blob/master/ci/maven.yml